### DISCUSSION TITLE
Zemismart ZM25TQ (id jzmy5ut0vishwscm) the state is displayed incorrectly

### DISCUSSION THREAD

**Discussion by Dexter-Lucky** [make-all/tuya-local#1823] [Q&A]
Hi!
I'm not sure if it's a bug or my mistake
I've got curtain motor Zemismart ZM25TQ (id jzmy5ut0vishwscm)
No possibility to close the curtains from the extreme position. When the curtains are fully open, the state is recorded as 'closed' and the close button is inactive. Similarly, when fully closed.
Attaching  screenshots from HA and TUYA app (curtains is closed):
![image](https://github.com/make-all/tuya-local/assets/127134937/10fd5f19-1404-4add-9f97-d22ba4cf2880)
![image](https://github.com/make-all/tuya-local/assets/127134937/405f56d5-d69b-43a7-9e14-4a57517ec2a4)
![photo_2024-04-12_23-29-31](https://github.com/make-all/tuya-local/assets/127134937/e8a8c0e3-3dd0-4045-a9d9-a8a409537c6c)

Please help me fix this.

**Comment by make-all** [comment]
Looking at the history of the file, it originally had an invalid "invert" config for the position dps, which was fixed recently. But probably it was working correctly before the fix, so rather than correcting the invalid config the better fix is to remove it.

**Reply by Dexter-Lucky** [reply]
Yes, thank you!!!
Now it's work's correct!
